app/utils/callback/tests/test_lava_callback.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) Collabora Limited 2020
```

Comment:
Copyright 2021

Reply:
In the commit I see "parse'ing", but parsing is correct.

Reply:
corrected